pages/toolkits/productivity/jira.mdx

```diff
@@ -27,9 +27,20 @@ The Jira toolkit provides a comprehensive set of tools for interacting with Jira
 This toolkit streamlines the process of issue management, making it easier to integrate Jira functionalities into applications and workflows.
 
 <Warning>
-  **The Jira toolkit only supports users authenticated with a single Jira instance.**
+  ## Handling multiple Atlassian Clouds
 
-  When users are prompted to authorize access to their Jira account, they will be asked to select a Jira instance. If a user selects "Instance A" and later also authenticates with "Instance B", the toolkit will raise an exception.
+  A Jira user may have multiple Atlassian Clouds authorized. In such cases, the Jira tools must be called with the `atlassian_cloud_id` argument. The [`Jira.GetAvailableAtlassianClouds`](/toolkits/productivity/jira#jiragetavailableatlassianclouds) tool can be used to get the available Atlassian Clouds and their IDs.
+
+  When a tool call does not receive a value for `atlassian_cloud_id` and the user only has a single Atlassian Cloud authorized, the tool will use that. Otherwise, an error will be raised. The error will contain an additional content listing the available Atlassian Clouds and their IDs.
+
+  Your AI Agent or AI-powered chat application can use the tool referenced above (or the exception's additional content) to guide the user into selecting the correct Atlassian Cloud.
+
+  When the user selects an Atlassian Cloud, it may be appropriate to keep this information in the LLM's context window for subsequent tool calls, avoiding the need to ask the user multiple times.
+
+  **_It is the job of the AI Agent or chat application to:_**
+
+  1. Make it clear to the chat's end user which Atlassian Cloud is being used at any moment, to avoid, for example, having a Jira Issue being created in the wrong Atlassian Cloud;
+  1. Appropriately instruct the LLM and keep the relevant information in its context window, enabling it to correctly call the Jira tools.
```

Comment:
Maybe an example here?

Reply:
I think this depends too heavily on how the agent or chat application is implemented to give an example.

Do you have an idea for a generic example?